devops/sql/creditRisk.sql

```diff
@@ -0,0 +1,390 @@
+    email_id varchar(50) NOT NULL,
+    created_date DATE,
+    modified_date DATE,
+    is_deleted BOOLEAN,
+    CONSTRAINT FK_CLIENT_CONTACT
+        FOREIGN KEY(client_id)
+            REFERENCES CLIENT(client_id)
+);
+
+CREATE TABLE family_details
+(
+    family_details_id BIGSERIAL CONSTRAINT  PK_FAMILY_DETAILS_ID PRIMARY KEY,
+    client_id BIGINT NOT NULL,
+    marital_status VARCHAR(20) NOT NULL,
+    residential_status VARCHAR(20),
+    no_of_dependents BIGINT,
+    created_date DATE,
+    modified_date DATE,
+    is_deleted BOOLEAN,
+    CONSTRAINT FK_CLIENT_FAMILY
+        FOREIGN KEY(client_id)
+            REFERENCES CLIENT(client_id)
+);
+
+CREATE TABLE credit_application_status
+(
+    credit_application_status_id BIGSERIAL CONSTRAINT  PK_CREDIT_APPLICATION_STATUS_ID PRIMARY KEY,
+    client_id BIGINT NOT NULL,
+    status_date DATE,
+    is_rejected_before BOOLEAN,
+    status VARCHAR(20) NOT NULL,
+    status_comments VARCHAR(120),
+    created_date DATE,
+    modified_date DATE,
+    is_deleted BOOLEAN,
+    CONSTRAINT FK_CLIENT_CREDIT_STATUS
+        FOREIGN KEY(client_id)
+            REFERENCES CLIENT(client_id)
+);
+
+
+CREATE TABLE credit_information
+(
+    credit_information_id BIGSERIAL CONSTRAINT  PK_CREDIT_INFORMATION_ID PRIMARY KEY,
+    client_id BIGINT NOT NULL,
+    credit_amount DECIMAL(19,2),
+    interest_rate DECIMAL(19,2),
+    planned_emi DECIMAL(19,2),
+    duration_in_months BIGINT NOT NULL,
+    created_date DATE,
+    modified_date DATE,
+    is_deleted BOOLEAN,
+    CONSTRAINT FK_CLIENT_CREDIT_INFORMATION
+        FOREIGN KEY(client_id)
+            REFERENCES CLIENT(client_id)
+);
+
+
+CREATE TABLE address_details
+(
+    address_details_id BIGSERIAL CONSTRAINT  PK_ADDRESS_DETAILS_ID PRIMARY KEY,
+    client_id BIGINT NOT NULL,
+    house_no BIGINT NOT NULL,
+    street_name VARCHAR(50) NOT NULL,
+    city VARCHAR(50) NOT NULL,
+    postal_code BIGINT NOT NULL,
+    created_date DATE,
+    modified_date DATE,
+    is_deleted BOOLEAN,
+    CONSTRAINT FK_CLIENT_ADDRESS
+        FOREIGN KEY(client_id)
+            REFERENCES CLIENT(client_id)
+);
+
+
+CREATE TABLE financial_information
+(
+    financial_information_id BIGSERIAL CONSTRAINT  PK_FINANCIAL_INFORMATION_ID PRIMARY KEY,
+    client_id BIGINT NOT NULL,
+    employment_status VARCHAR(20) NOT NULL,
+    annual_income DECIMAL(19,2) NOT NULL,
+    employed_years BIGINT,
+    position VARCHAR(20),
+    company_employed VARCHAR(20),
+    created_date DATE,
+    modified_date DATE,
+    is_deleted BOOLEAN,
+    CONSTRAINT FK_CLIENT_FINANCIAL_INFORMATION
+        FOREIGN KEY(client_id)
+            REFERENCES CLIENT(client_id)
+);
+
+--Client
+INSERT INTO client (client_id, first_name, last_name, date_of_birth, social_security_number, gender, citizenship, existing_client_id, created_date, modified_date, is_deleted) VALUES (1, 'Judy', 'Green', '1970-11-03', '377-82-5154', 'M', '2021-03-08 15:50:16.111321', null, '2021-03-08', null, false);
```

Comment:
Would it be possible to use the same structure for INSERT as in accounts db, and list values only instead of having many INSERT instruction? This looks very redundant and hard to read the actual values in text editor.

```
INSERT INTO CLIENT(clientId, firstName, lastName, socialSecurityNumber, sex, dateOfBirth, createdDate,citizenship,flagged_customer)
VALUES
( 'B12456'	,'Brandi'	    ,'Savage'   ,'7854' ,'F','1995-11-18',CURRENT_DATE ,'Austria'  ),
( 'C22466'	,'Christopher'	,'Arnold'   ,'3049' ,'M','1978-11-18',CURRENT_DATE ,'Nambia'  ),
( 'J12578'  ,'Jerry'        ,'Miranda'  , '4164','F','1970-11-18',CURRENT_DATE ,'Austria' ),
```

Reply:
Yup, ofcourse. I just used the system generated scripts and that is why they are redundant